lib/interactive_viewer.ml

```diff
@@ -52,20 +46,47 @@ let current_operation patches =
   | Some p -> ui_of_operation p.Patch.operation
   | None -> W.string "No operation"
 
+let calculate_add_remove lines =
+  List.fold_left
+    (fun (add, rem) line ->
+      match line with
+      | `Their _ -> (add + 1, rem)
+      | `Mine _ -> (add, rem + 1)
+      | _ -> (add, rem))
+    (0, 0) lines
+
+let modification_counter patches =
+  let$ current_patch = current_patch patches in
+  match current_patch with
+  | Some p ->
+      let hunks = p.Patch.hunks in
+      let additions, removals =
+        List.fold_left
+          (fun (add_acc, remove_acc) hunk ->
+            let addition, removal = calculate_add_remove hunk.Patch.lines in
+            (add_acc + addition, remove_acc + removal))
+          (0, 0) hunks
```

Comment:
Suggestion: move this and `calculate_add_remove` (renamed to `calculate_add_remove_hunk`) into a function named `calculate_add_remove_operation`. It makes it clearer what function does what!